airbyte-integrations/connectors/source-clockify/source_clockify/streams.py

```diff
@@ -0,0 +1,120 @@
+
+from abc import ABC
+from typing import Any, Iterable, Mapping, MutableMapping, Optional
+
+from airbyte_cdk.models import SyncMode
+from airbyte_cdk.sources.streams.http import HttpStream
+
+import requests
+
+
+# Basic full refresh stream
+class ClockifyStream(HttpStream, ABC):
+    url_base = "https://api.clockify.me/api/v1/"
+    page_size = 3
+    page = 1
+    primary_key = None
+
+    def __init__(self, workspaceId: str, **kwargs):
+        super().__init__(**kwargs)
+        self.workspaceId = workspaceId

```

Comment:
```suggestion
        self.workspace_id = workspaceId

```

Please check Python PEP-8 recommendations for variable naming.